tests/helpers.jsx

```diff
@@ -172,6 +194,7 @@ module.exports = {
       },
       UserStore: {
```

Comment:
so the common way is to pass a flux instance from top to bottom. and have a utility for the flux instance to connect to a store, which extracts out data to use as props for an inner component.

something like https://github.com/acdlite/flummox/blob/master/docs/docs/api/fluxcomponent.md